cpp/src/parquet/properties.h

```diff
@@ -186,6 +200,24 @@ class PARQUET_EXPORT ColumnProperties {
     page_index_enabled_ = page_index_enabled;
   }
 
+  void set_bloom_filter_options(std::optional<BloomFilterOptions> bloom_filter_options) {
+    if (bloom_filter_options) {
+      if (bloom_filter_options->fpp > 1.0 || bloom_filter_options->fpp < 0.0) {
+        throw ParquetException(
+            "Bloom Filter False positive probability must be between 0.0 and 1.0");
+      }
+    }
+    bloom_filter_options_ = bloom_filter_options;
+  }
+
+  void set_bloom_filter_enabled(bool bloom_filter_enabled) {
```

Comment:
Do you mean that we can support interface like:

```c++
void enable_and_set_bloom_filter_options(BloomFilterOptions bloom_filter_options);
void disable_bloom_filter(BloomFilterOptions bloom_filter_options);
```

?

Reply:
Yeah I agree.( I still think we should add a disable here for stable testing or config it in Python and other SDK)

And I agree `enable_bloom_filter` is not a good idea. Currently user should better have a self-given ndv rather than a default huge value